### PR TITLE
`setX` -> `setY`; `setM` -> `setX`

### PR DESCRIPTION
**Pull request by affeldt-aist**
##### Motivation for this change

@CohenCyril @t6s @proux01 @Tragicus 

##### Checklist

- [x] added corresponding entries in `CHANGELOG_UNRELEASED.md`

- [x] added corresponding documentation in the headers

Reference: [How to document](https://github.com/math-comp/math-comp/wiki/How-to-document)

##### Reminder to reviewers

- Read [this Checklist](https://github.com/math-comp/math-comp/wiki/Checklist-for-creating-and-review-PRs)
- Put a milestone if possible
- Check labels

### CODE REVIEW

File: CHANGELOG_UNRELEASED.md [hunk 1]
```diff
@@ -13,12 +13,13 @@
   + factory `isAlgebraOfSets_setD`
 
 - in `classical_sets.v`:
-  + definition `setX`, notation ``` `^` ```
-  + lemmas `setX0`, `set0X`, `setXK`, `setXC`, `setXA`, `setIXl`, `mulrXr`,
-    `setX_def`, `setXE`, `setXU`, `setXI`, `setXD`, `setXCT`, `setCXT`, `setXTC`, `setTXC`
+  + definition `setY`, notation ``` `^` ```
```
Comment (CohenCyril) [inline CHANGELOG_UNRELEASED.md:16]: I am also going to argue that `^` generally means exponential and I would rather associate notation ``A `^` B`` to `set_fun A B`. For `setY` maybe we can pick `y` for now?

**Review by CohenCyril — CHANGES_REQUESTED** [review]

File: CHANGELOG_UNRELEASED.md (hunk 1 above)
Comment (affeldt-aist) [reply CHANGELOG_UNRELEASED.md:16]: Or ``` `+` ``` ?

Comment (proux01) [reply CHANGELOG_UNRELEASED.md:16]: FWIW, the notation for exclusive or in ssrbool is `_ (+) _`

**Comment by affeldt-aist** [comment]
I went for ``` `+` ``` because it is shorter. @CohenCyril

**Pull request merged**